Src/LinearSolvers/MLMG/AMReX_MLTensorOp.cpp

```diff
@@ -211,6 +211,11 @@ MLTensorOp::apply (int amrlev, int mglev, MultiFab& out, MultiFab& in, BCMode bc
     applyBCTensor(amrlev, mglev, in, bc_mode, s_mode, bndry );
 
     const auto dxinv = m_geom[amrlev][mglev].InvCellSizeArray();
+    const Geometry& geom = m_geom[amrlev][mglev];
+    const Box& domain = amrex::enclosedCells(geom.Domain());
```

Comment:
The call to `enclosedCells` here and other places is actually unnecessary because `Geometry::Domain()` is always cell-centered.